.github/workflows/upload-framework.yml

```diff
@@ -59,4 +54,4 @@ jobs:
       - name: Upload Release Framework
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: gh release upload ${{ steps.release.outputs.tag_name }} ./FrameworkBuild/Bucketeer.xcframework.zip
+        run: gh release upload ${GITHUB_REF#refs/*/} ./FrameworkBuild/Bucketeer.xcframework.zip
```

Comment:
Shouldn't we use `steps.release.outputs.tag_name` instead?

Reply:
I understand that `tag_name` is a variable that I can use because the [GitHub Actions step](https://github.com/google-github-actions/release-please-action?tab=readme-ov-file#root-component-outputs)(`id:release`) I was using before the change exists.

I think tag_name cannot be used because this modification deleted that step(`id:release`), but am I misunderstood?

It seems that I can get the tag using `GITHUB_REF`.
https://stackoverflow.com/questions/58177786/get-the-current-pushed-tag-in-github-actions